roles/serviceassurance/tasks/_local_signing_authority.yml

```diff
@@ -7,77 +7,32 @@
       apiVersion: certmanager.k8s.io/v1alpha1
       kind: Issuer
       metadata:
-        name: '{{ meta.namespace }}-selfsigned'
+        name: elasticsearch-selfsigned
```

Comment:
Does the issuer certificate need to be called "elasticsearch" for some reason, or could we keep a single generic issuer for the namespace and use it to issue certs for various purposes?

Also, this file seems somewhat ill-named if it's dedicated to ES only.

Reply:
I will create a generic one. Good point.

Reply:
Fixed. Let me know if that makes sense.